app/src/main/res/layout/lessons_chapter_view.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file shouldn't be introduced--did you accidentally reintroduce it due to a recent merge? (Or maybe I did?)

Reply:
Yeah Ben had introduced it trying to solve merge conflicts, getting rid of it since it is not required in this branch.